langserver/build_context.go

```diff
@@ -10,12 +9,15 @@ import (
 	"path/filepath"
 	"runtime"
 	"strings"
+
+	"golang.org/x/net/context"
 )
 
-func (h *LangHandler) defaultBuildContext() *build.Context {
-	bctx := &build.Default
+// BuildContext creates a build.Context which uses the overlay FS and the InitializeParams.BuildContext overrides.
+func (h *LangHandler) BuildContext(ctx context.Context) *build.Context {
+	var ctxt *build.Context
```

Comment:
nit: doesn't the old `bctx` var name make more sense given `ctx` is already in scope here from the parameter?